CONTRIBUTING.md

```diff
@@ -0,0 +1,418 @@
+# Contributing to knative/docs
+
+- [Before you begin](#before-you-begin)
+- [Contributing to the documentation](#contributing-to-the-documentation)
+- [Docs contributor roles](#docs-contributor-roles)
+
+**First off, thanks for taking the time to contribute!**
+
+The following are guidelines for contributing to the Knative documentation.
+These are just guidelines, not rules. Use your best judgment, and feel free to
+propose changes to this document in a pull request.
+
+## Before you begin
+
+### Code of conduct
+
+Knative follows the
+[Knative Code of Conduct](https://github.com/knative/community/blob/master/CODE-OF-CONDUCT.md).
+By participating, you are expected to uphold this code. Please report
+unacceptable behavior to knative-code-of-conduct@googlegroups.com.
+
+### Contributor license agreements
+
+We'd love to accept your contributions! But before we can take them, you need to
+fill out the [Google CLA](https://cla.developers.google.com). Important: You
+must fill out the CLA with the same email address that you used to create your
+GitHub account.
+
+Once you are CLA'ed, we'll be able to accept your pull requests. This is
+necessary because you own the copyright to your changes, even after your
+contribution becomes part of this project. So this agreement simply gives us
+permission to use and redistribute your contributions as part of the project.
+
+Note: Your contributions are verified using the email address for which you use
+to sign the CLA. Therefore,
+[setting your GitHub account to private](https://help.github.com/en/articles/setting-your-commit-email-address)
+is unsupported because all commits from private accounts are sent from the
+`noreply` email address.
+
+### Style guide
+
+Knative documentation follows the
+[Google Developer Documentation Style Guide](https://developers.google.com/style/);
+use it as a reference for writing style questions.
+
+## Contributing to the documentation
+
+### Reporting documentation issues
+
+Knative uses Github issues to track documentation issues and requests. If you
+see a problem with the documentation, submit an issue using the following steps:
+
+1. Check the [Knative docs issues list](https://github.com/knative/docs/issues)
+   before creating an issue to avoid making a duplicate.
+2. Use the [correct template](https://github.com/knative/docs/issues/new) for
+   your new issue. There are two templates available:
+   - **Bug report**: If you're reporting an error in the existing documentation,
+     use this template. This could be anything from broken samples to typos.
```

Comment:
What is a `broken sample`? Should this maybe be `broken link` instead?

Reply:
A broken sample would presumably be one of the code samples which no longer compiles or works properly due to ecosystem changes.

Reply:
A broken sample would presumably be one of the code samples which no longer compiles or works properly due to ecosystem changes.